types/k6/experimental/timers.d.ts

```diff
@@ -27,7 +27,7 @@ export function clearTimeout(timeoutID: TimeoutID): void;
  * @param args - The arguments to be passed to the function.
  * @returns The interval id.
  */
-export function setInterval(functionRef: (...args: any[]) => void, delay: number, ...args: any[]): void;
+export function setInterval(functionRef: (...args: any[]) => void, delay: number, ...args: any[]): number;
```

Comment:
```suggestion
export function setInterval(functionRef: (...args: any[]) => void, delay: number, ...args: any[]): IntervalID;
```

it not only is a number, but it specifically an IntervalID and only it can be used in the `clearInterval` call which is likely why you want this change in the first place